src/lib/deeplink/handlers/add-site-blueprint-with-url.ts

```diff
@@ -92,15 +74,18 @@ export async function handleAddSiteWithBlueprint( urlObject: URL ): Promise< voi
 
 		await sendIpcEventToRenderer( 'add-site-blueprint-from-url', { blueprintPath } );
```

Comment:
What do you think about renaming the event name to `add-site-with-blueprint-path` or simpler `add-site-with-blueprint`?

Reply:
If we do then we probably need to rename filename (`add-site-blueprint-with-url.ts`) as well